src/Features/Core/Portable/CodeRefactorings/SyncNamespace/AbstractChangeNamespaceService.cs

```diff
@@ -434,13 +434,29 @@ private static async Task<Solution> MergeDocumentChangesAsync(Solution originalS
             return originalSolution;
         }
 
-        private static async Task<ImmutableArray<ReferenceLocation>> FindReferenceLocationsForSymbol(
+        private struct LocationForAffectedSymbol
```

Comment:
readonly struct.

Reply:
nit: this type is *almost* basic enough that i would be fine with jsut using a tuple.  However, if you want to keep this type, i'm also ok with it.

Reply:
Yea, I started with a tuple, but the length of declaration feels longer than I'd like, so I switched to a struct. C# 8 records would be handy here.